crates/holochain/src/conductor/api/api_external/app_interface.rs

```diff
@@ -65,14 +65,16 @@ impl AppInterfaceApi for RealAppInterfaceApi {
                 self.conductor_handle.get_app_info(&app_id).await?,
             )),
             AppRequest::SignalSubscription(_subscription) => {
-                todo!("Signal pubsub not yet implemented")
+                unimplemented!("Signal pubsub currently unimplemented")
```

Comment:
This is just to make it consistent with AppRequest::Crypto variant (unimplemented) below